Project_Mars/Project_Mars/Pages/LanguageTab.cs

```diff
@@ -0,0 +1,270 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.DevTools;
+using OpenQA.Selenium.DevTools.V117.Debugger;
+using OpenQA.Selenium.Support.UI;
+using Project_Mars.Utilities;
+using SeleniumExtras.WaitHelpers;
+using System.Data.SqlTypes;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+
+namespace Project_Mars.Pages
+{
+    public class LanguageTab
+    {
+        private readonly IWebDriver driver;
+
+        //public LanguageTab(IWebDriver driver)
+        //{
+        //    this.driver = driver;
+        //}
+
+        public LanguageTab()
+        {
+        }
+
+        public void CreateLanguageRecord(IWebDriver driver, string LanguageName, string LanguageLevel)
+        {
+            Wait.WaitToBeClickable(driver, "XPath", "//div[@data-tab='first']//div[@class ='ui teal button ']", 10);
+
+            int totalrows = driver.FindElements(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table//tbody")).Count;
+            Console.WriteLine(totalrows);
+
+            if (totalrows < 4)
```

Comment:
here you should not be doing this, instead you should be reading in a list of elements in the table see https://www.guru99.com/handling-dynamic-selenium-webdriver.html, also you need to think about state, 
the state is say that you have a test that is adding a language english, the second time you run this test you should delete any existing languages so that the state is reset, this will guarantee that your test does not produce false negatives and passes always please fix this

Reply:
Added test case to delete all existing languages 
![image](https://github.com/Shilpii01/Project_Mars/assets/145233624/df2f9894-33ca-4995-ac2a-3d858050f6ff)